### ISSUE TITLE
rank loss 的 taget 似乎没有起作用？

### ISSUE THREAD

**Issue by yichengsu** [zheng-yuwei/RankIQA.PyTorch#8]
看上去`target`总是等于`torch.ones(args.batch_size)`，似乎没有起到作用？

https://github.com/zheng-yuwei/RankIQA.PyTorch/blob/fad8c07d7b45e59b1384cdb0a31724a395ee3a86/criterions/ranking_loss.py#L27-L28

**Comment by STAR-811** [comment]
> 看上去`target`总是等于`torch.ones(args.batch_size)`，似乎没有起到作用？
> 
> https://github.com/zheng-yuwei/RankIQA.PyTorch/blob/fad8c07d7b45e59b1384cdb0a31724a395ee3a86/criterions/ranking_loss.py#L27-L28

前面一张图的分数比后面的高，第三个参数就应该是1，而一共有batch size对图，所以是torch.ones(args.batch_size)
